internal/cli/require/require.go

```diff
@@ -49,6 +50,17 @@ func ExactArgs(n int) cobra.PositionalArgs {
 	}
 }
 
+// ExactArgsObjectID returns an error if there are not exactly n args and
+// any of those args is not an ObjectID.
+func ExactObjectIDArgs(n int) cobra.PositionalArgs {
+	return func(cmd *cobra.Command, args []string) error {
+		if err := ExactArgs(n)(cmd, args); err != nil {
+			return err
+		}
+		return objectIDArgs(args)
+	}
+}
+
```

Comment:
[q] Could this function takes `func objectIDArgs()` as input? so we won't have to define the same method if we want to apply other validations to the args.

Reply:
At this point that would be a premature abstraction, specially if you mean here is to have something like `ExactCustomValidArgs(n int, f func([]string) error) cobra.PositionalArgs` and specially as you can already do that at the cobra level, we may just have enought of a excuse to make the objectID validation aa thing as many commands will take an objectID. What other custom validations you see here?

Reply:
I wasn’t thinking about any particular validation but I was wondering if this change could make sense here. Thanks for the answer!